modules/smart-agent_cassandra-nodetool/conf/01-state.yaml

```
@@ -0,0 +1,14 @@
+## Example
+module: cassandra nodetool
+name: "Node is not in normal status"
+
+transformation: ".min(over='10m')"
+aggregation: false
```

I would set it to true to give the user ability to cusotmize this from variable if needed:

```suggestion
aggregation: true
```